redis/client.py

```diff
@@ -1411,6 +1432,20 @@ def get_message(self, ignore_subscribe_messages=False, timeout=0):
         before returning. Timeout should be specified as a floating point
         number.
         """
+        if not self.subscribed:
+            # Wait for subscription
+            start_time = time.time()
+            if self.subscribed_event.wait(timeout) is True:
```

Comment:
```suggestion
            if self.subscribed_event.wait(timeout):
```

Reply:
@chayim asked to make statements as explicit as possible